review-tools/ghmerge

```diff
@@ -59,7 +60,11 @@ while [ $# -ne 0 ]; do
         WHAT=web ; BUILD=no ; shift
         ;;
     --cherry-pick)
-        PICK=yes ; shift
+        shift;
+        PICK=1;
+        if [ "$1" != "" ] && [ $1 -ge 1 ] && [ $1 -le 99 ]; then
+            PICK=$1 ; shift
+        fi
```

Comment:
you need to do:
`if [ "$1" != "" ] && [ $1 -ge 1 ] 2>/dev/null && [ $1 -le 99 ]; then`

Reply:
Ah, I had not seen your hint regarding `2>/dev/null`, done now.